handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+# MongoDB
+
+MongoDB is used by the majority of SystemLink services as the primary datastore. MongoDB Community Edition is included in the default installation of SystemLink will launch when the SystemLink server boots.
+
+!!!note
+    By default SystemLink stores all data on databases and the file system hosted on the SystemLink application server. This is referred to as a *single node* deployment. To improve the reliability of data storage and scale to larger volumes of data, remote data stores are recommended for all production deployments. This is referred to as a *multi node* deployment.
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink 2021R1 or greater. Refer to Installing and Configuring SystemLink Server and Clients for the basics of setting up a SystemLink server.
+- A standalone server running MongoDB, multiple servers hosting a MongoDB replica set, or a MongoDB Atlas Account with a user with the `dbAdmin` or similar role. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details.
+
+### SystemLink with Single Node vs Multi Node MongoDB Deployments
+
+The following table summarizes when to consider the various supported configurations of MongoDB with SystemLink.
+
+| Diagram | SystemLink + MongoDB Configuration | When to use |
+| | ----------- | ----------- |
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with Defaults     | Here SystemLink and MongoDB are hosted on the same server. Use this configuration when evaluating SystemLink or in small deployments of less than 10 managed nodes without the Asset Manager and Test Insights modules. |
```

Comment:
For each of these "When to use" descriptions, more user-centered language and bulleted lists could help make this clearer and more scannable.

So here, that could look something like:

- You have one server hosting both MongoDB and SystemLink
- You're evaluating SystemLink
- You are working with less than 10 managed nodes
- You do not need the Asset Manager or Test Insights modules.

Reply:
I agree and sadly we can't do bulleted lists in tables in markdown 😫. Regardless of this constraint I bet these paragraphs can be cleaned up and made more succinct.